lib/beaker/host_prebuilt_steps.rb

```diff
@@ -411,8 +411,10 @@ def enable_root_login host, opts
       block_on host do |host|
         logger.debug "Update /etc/ssh/sshd_config to allow root login"
```

Comment:
nit: this should be updated to just `sshd_config` in the debug message, as the location varies per version.